src/Components/Samples/BlazorServerApp/wwwroot/BlazorServerApp.lib.module.js

```diff
@@ -0,0 +1,7 @@
+export function beforeStart(serverOptions) {
+    console.log('starting!');
+}
+
+export function afterStarted(blazor) {
+    console.log('started!');
+}
```

Comment:
Would it be OK to not add this file? Generally the "playground" apps (of which this is one) get really hard to use if they become filled up with examples of random features. Where possible it's great to keep this project and the WebAssembly standalone project close to the project template.

Reply:
absolutely, just did it to make sure I could test things (since I had to monkeypatch my sdk to do this) and there are also E2E test for this.